src/components/fx/hover.js

```diff
@@ -1063,14 +1063,13 @@ function cleanPoint(d, hovermode) {
     d.y0 = Lib.constrain(d.y0, 0, d.ya._length);
     d.y1 = Lib.constrain(d.y1, 0, d.ya._length);
 
-    // and convert the x and y label values into objects
-    // formatted as text, with font info
+    // and convert the x and y label values into formatted text
     if(d.xLabelVal !== undefined) {
-        d.xLabel = ('xLabel' in d) ? d.xLabel : Axes.hoverLabelText(d.xa, d.xLabelVal);
+        d.xLabel = ('xLabel' in d) ? d.xLabel : getDimText(d, 'x');
```

Comment:
@etpinard I made `getDimText` in parallel with you adding `d.(x|y)Label` and `Axes.hoverLabelText` - I suppose in principle we could use `d.(x|y)Label` for this too instead of adding `d.(x|y)LabelVal(0|1)`, maybe that would actually be simpler... let me take a look.

Reply:
yeah, better to use `d.(x|x)Label` -> https://github.com/plotly/plotly.js/pull/2113/commits/51ad8c239653ba0bb69387cefd791409fa6208db